neo/rawio/spikeglxrawio.py

```diff
@@ -160,6 +161,20 @@ def _parse_header(self):
                 stream_name = signal_stream['name']
                 sig_ann = self.raw_annotations['blocks'][0]['segments'][seg_index]['signals'][c]
 
+                if self.load_channel_location:
```

Comment:
Maybe this is something we should expose on the non-rawio level? There we could expose the PI probe directly without indirect handling via annotations.

Reply:
Yes we could but this is outside this because it would be more general.
The SpikeGLXIO also have this kwargs.

Reply:
ok, lets keep it here then.